### DISCUSSION TITLE
Will drift support cloudkit?

### DISCUSSION THREAD

**Discussion by mny459** [simolus3/drift#1953] [Q&A] [answered]
As title

**Comment by simolus3** [comment, marked as answer]
I don't have any plans for supporting CloudKit. Drift is designed around relational SQL databases. CloudKit is kind of relational, but it's API is not nearly versatile enough to run (or even efficiently emulate running) SQL queries ontop of it.

**Reply by mny459** [reply]
Got it. Think you reply
